stubs/requests/requests/sessions.pyi

```diff
@@ -67,12 +67,14 @@ _Params: TypeAlias = Union[
     str | bytes,
 ]
 _TextMapping: TypeAlias = MutableMapping[str, str]
+_HeadersMapping: TypeAlias = MutableMapping[str, str] | MutableMapping[str, bytes]
+_HeadersUpdateMapping: TypeAlias = MutableMapping[str, str] | MutableMapping[str, bytes] | MutableMapping[str, None]
```

Comment:
I'm not sure that's correct, header values can be some str and some bytes?

Reply:
Mmm I see - would we have to specify all combinations of mutable mappings for str, bytes and None?

Reply:
b5ec834